<!--离线命令词识别-->
<launch>
  <!--设置为功能包的路径-->
  <arg name="package_path" default = "$(find xf_mic_asr_offline)" />

  <node pkg="xf_mic_asr_offline" type="xf_asr_offline_node" name="xf_asr_offline_node" output="screen">
    <param name="source_path" type="string" value="$(arg package_path)"/>
    <rosparam command="load" file="$(find xf_mic_asr_offline)/config/mic_offline_params.yaml" />
  </node>
  
</launch>
